samples/cpp/benchmark_app/main.cpp

```diff
@@ -80,7 +80,6 @@ bool parse_and_check_command_line(int argc, char* argv[]) {
     }
 
     if (FLAGS_latency_percentile > 100 || FLAGS_latency_percentile < 1) {
-        show_usage();
```

Comment:
Can you put it back
Relates to https://github.com/openvinotoolkit/openvino/pull/12814#discussion_r1016665661

Reply:
Oki, I'll add it to Python instead

Reply:
I've noticed this check exists already: `if parsed_args.latency_percentile < 0 or parsed_args.latency_percentile > 100:
        parser.print_help()
        raise RuntimeError("The percentile value is incorrect. The applicable values range is [1, 100].")` However, is '0' a valid value here?

Reply:
It's already there. https://github.com/openvinotoolkit/openvino/blob/6f2eab44133dc84170fdf4dd10f32517e67cc7fe/tools/benchmark_tool/openvino/tools/benchmark/parameters.py#L161

But I didn't notice there's another place for such checks in python. Would be good, if you move it

Reply:
> However, is '0' a valid value here? I'll set this check to `if parsed_args.latency_percentile < 0 or parsed_args.latency_percentile > 100` to keep it consistent with the message

It looks 0 should be prohibited